api/src/indigo_misc.cpp

```diff
@@ -1334,6 +1334,7 @@ CEXPORT const char* indigoCheckStructure(const char* structure, const char* prop
     INDIGO_END(0);
 }
 
+/*
 CEXPORT const char* indigoJson(int item)

```

Comment:
Better remove this function completely for clarity, not to lid the code with commented stuff.